setup.cfg

```
@@ -12,50 +12,41 @@ classifiers =
   Development Status :: 4 - Beta
   Operating System :: MacOS :: MacOS X
   Operating System :: POSIX :: Linux
-  Programming Language :: Python
+  License :: OSI Approved :: MIT License
   Programming Language :: Python :: 3
-  Programming Language :: Python :: 3.14
-  Programming Language :: Python :: 3.13
   Programming Language :: Python :: 3.12
   Programming Language :: Python :: 3.11
   Programming Language :: Python :: 3.10
   Programming Language :: Python :: 3.9
-  Programming Language :: Python :: 3.8
-  Programming Language :: Python :: 3.7
-  Programming Language :: Python :: 3.6
-  Programming Language :: Python :: 3.5
-  Programming Language :: Python :: 3.4
-  Programming Language :: Python :: 3.3
-  Programming Language :: Python :: 2.7
+  Programming Language :: Python :: 3 :: Only
   Topic :: Software Development :: Libraries :: Python Modules
-  Topic :: Security
+  Topic :: System :: Networking
 license = MIT
 license_files =
-  LICENSE[.md]*
+  LICENSE.md
 
 platform = any
 project_urls =
-  Bug Tracker = https://github.com/reactive-firewall/python-repo/issues
+  "Bug Tracker" = https://github.com/reactive-firewall/python-repo/issues
   License = https://github.com/reactive-firewall/python-repo/LICENSE.md
+  Documentation = https://reactive-firewallpython-repo.readthedocs.io/en/stable
+  Repository = https://github.com/reactive-firewall/python-repo.git
 
 [bdist_rpm]
 url = https://github.com/reactive-firewall/python-repo.git
 
 [bdist_wheel]
-universal=1
-
-[files]
-packages = pythonrepo
+universal=0
 
 [options]
-zip_safe = true
+zip_safe = false
 py_modules = pythonrepo
 test_suite = tests
-python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.15.*
+python_requires = >=3.9.20
 setup_requires =
-  setuptools>=45.0.0
-  wheel>=0.37.0
-  build>=1.1.1
+  setuptools>=75.0.0
+  wheel>=0.44.0
+  build>=1.2.1
```

Revert this to
```suggestion
  setuptools >=38.0.0
  wheel>=0.35.0
```

an in requirements.txt need to use:

```plain
setuptools==38.0.0; python_version >= '2.7' and python_version < '3.5'
setuptools==45.0.0; python_version >= '3.5' and python_version <= '3.9.19'
setuptools>=75.0.0; python_version >= '3.9.20'
wheel>=0.35.0; python_version >= '2.7' and python_version <= '3.7'
wheel>=0.44.0; python_version >= '3.8'
build>=1.2.1; python_version >= '3.9.20'
```